library/src/com/google/maps/android/geojson/GeoJsonPointStyle.java

```diff
@@ -0,0 +1,297 @@
+package com.google.maps.android.geojson;
+
+import com.google.android.gms.maps.model.BitmapDescriptor;
+import com.google.android.gms.maps.model.MarkerOptions;
+
+import java.util.Observable;
+
+/**
+ * A class that allows for GeoJsonPoint objects to be styled and for these styles to be translated
+ * into a MarkerOptions object
+ */
+public class GeoJsonPointStyle extends Observable implements GeoJsonStyle {
+
+    private final static String GEOMETRY_TYPE_REGEX = "Point|MultiPoint|GeometryCollection";
+
+    private final MarkerOptions mMarkerOptions;
+
+
+    /**
+     * Creates a new PointStyle object
+     */
+    public GeoJsonPointStyle() {
+        mMarkerOptions = new MarkerOptions();
+    }
+
+    /**
+     * Gets the type of geometries this style can be applied to
+     *
+     * @return type of geometries this style can be applied to
+     */
+    @Override
+    public String getGeometryType() {
```

Comment:
Is this expected to be used outside of this library? If not, drop the privacy to package-private or something appropriate. Also, you can use Pattern instead of String if the value truly represents a regex pattern.